.github/workflows/node.js.yml

```diff
@@ -4,35 +4,37 @@
 name: Node.js CI
 
 on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+    push:
```

Comment:
I can't see eslint rules here, but it should keep consistency with PixiJS repo https://github.com/pixijs/pixijs/blob/dev/.github/workflows/nodejs.yml 

They are using tab space 2